static/elements/chromedash-process-overview.js

```diff
@@ -138,6 +142,18 @@ class ChromedashProcessOverview extends LitElement {
          </tr>
        `)}
      </table>
+
+    ${Object.keys(this.progress).length ? html`
+      <chromedash-callout
+        cue="progress-checkmarks" targetid="progress-header" signedin
```

Comment:
Is the value for `signedin` missing intentionally?

Reply:
That's a shorthand for a boolean property being true, but I added ="true" to make it more explicit.